webapp/src/utils/jira_issue_metadata.tsx

```diff
@@ -264,6 +276,10 @@ export function isLabelField(field: JiraField | FilterField): boolean {
     return field.schema.system === 'labels' || field.schema.custom === 'com.atlassian.jira.plugin.system.customfieldtypes:labels';
 }
 
+export function isUserField(field: JiraField | FilterField): boolean {
+    return field.schema.type === 'user' || field.schema.custom === 'com.atlassian.jira.plugin.system.customfieldtypes:userpicker';
```

Comment:
Could the string 'com.atlassian.....' be a constant more to the top of the file? It seems some kind of "magic string".